spec/write_spec.rb

```diff
@@ -37,10 +38,23 @@
            tag_list: %w(baz bar),
            id: 200,
            position: 2,
-           choices: [{value: 'foo', checked: true}, {value: 'bar', checked: false}],
            type: 'problem',
            layout: 'input_right',
-           test: 'foo bar'}]) }
+           test: 'foo bar'},
+
+          {description: 'a multiple',
+           name: 'multiple',
+           tag_list: %w(mult),
+           id: 300,
+           editor: 'multiple_choice',
+           position: 3,
+           type: 'problem',
+           layout: 'input_right',
+           choices: [{'value' => 'foo', 'checked' => true}, {'value' => 'bar', 'checked' => false}]
```

Comment:
I think you will need to change atheneum guide import. It is expecting a list of strings instead of a list of objects. We need to map them in atheneum to a list of strings.